app/assets/javascripts/cascade/new-home.js

```diff
@@ -0,0 +1,1195 @@
+
+			// Attach function to content
+			this.$pageWrapper.delegate("a","click",function(e) {
+
+				var modifierKey = (e.metaKey || e.ctrlKey);
+
+				var $quickviewHTML = jQuery(e.currentTarget).attr('data-quickview-content');
+
+				var toggleAudio = jQuery(e.currentTarget).hasClass('toggleAudio');
+
+				if (!modifierKey && toggleAudio) {
+					cu_hero_area.toggleSound();
+				}
+
+				//Do not intercept URLs that are alt clicked or in small windows
+				if (!modifierKey && (cu_window_manager.windowWidth > 640)) {
+					heroQuickView.showContent($quickviewHTML);
+				}
+
+			});
+
+			// Close on click
+			this.$container.click(heroQuickView.hide);
+
+			// Close on esc key
+			jQuery('body').keyup(function(e) {
+				if (e.which == 27) heroQuickView.hide();
+			});
+		},
+
+		showContent : function(content) {
+			if (content.length > 0) {
+				heroQuickView.$containerCell.append(content).css('height',jQuery(window).height()+"px").css('width',jQuery(window).width()+"px");
+
+				heroQuickView.$container.fadeIn(500);
+
+				// lock scroll position, but retain settings for later
+				heroQuickView.lockScroll();
+
+				$('body').addClass('heroQuickview');
+			}
+		}, 
+
+		hide : function() {
+			heroQuickView.$container.fadeOut(40);
+			setTimeout(function(){
+				heroQuickView.$containerCell.empty();
+			},40);
+
+			// un-lock scroll position
+			heroQuickView.unlockScroll();
+
+			$('body').removeClass('heroQuickview');
+		},
+
+		lockScroll : function() {
+			// lock scroll position, but retain settings for later
+			var scrollPosition = [
+			  self.pageXOffset || document.documentElement.scrollLeft || document.body.scrollLeft,
+			  self.pageYOffset || document.documentElement.scrollTop  || document.body.scrollTop
+			];
+			var html = jQuery('html'); // it would make more sense to apply this to body, but IE7 won't have that
+			html.data('scroll-position', scrollPosition);
+			html.data('previous-overflow', html.css('overflow'));
+			html.css('overflow', 'hidden');
+			window.scrollTo(scrollPosition[0], scrollPosition[1]);
+			this.isScrollLocked = true;
+		}, // end lockScroll
+
+		unlockScroll : function() {
+			if (!this.isScrollLocked) return false;
+
+			var html = jQuery('html');
+			var scrollPosition = html.data('scroll-position');
+			html.css('overflow', html.data('previous-overflow'));
+			window.scrollTo(scrollPosition[0], scrollPosition[1]);
+		},
+
+		hasImageExtension : function(url) {
+			return(url.match(/\.(jpeg|jpg|gif|png)$/) != null);
+		}
+	} // end heroQuickView
+	heroQuickView.initialize();
+
+
+})(jQuery);
+
+// Define Lazybind
+(function($){
+	$.fn.lazybind = function(event, fn, timeout, abort){
+		var timer = null;
+		$(this).bind(event, function(e){
+			var ev = e;
+			timer = setTimeout(function(){
+				fn(ev);
+			}, timeout);
+		});
+		if(abort == undefined){
+			return;
+		}
+		$(this).bind(abort, function(){
+			if(timer != null){
+				clearTimeout(timer);
+			}
+		});
+	};
+})(jQuery);

```

Comment:
Is this `lazybind` or `delayedbind`? I'd be curious why this would be needed. Seems kludgey.